GitUI/Editor/FindAndReplaceForm.cs

```diff
@@ -401,11 +408,58 @@ protected override void Dispose(bool disposing)
 
             base.Dispose(disposing);
         }
+
+        internal TestAccessor GetTestAccessor() => new TestAccessor(this);
+
+        internal readonly struct TestAccessor
+        {
+            private readonly FindAndReplaceForm _control;
+
+            public TestAccessor(FindAndReplaceForm control)
+            {
+                _control = control;
+            }
+
+            public TestAccessor SetCriteria(string lookFor, bool matchCase = false, bool matchWholeWordOnly = false)
+            {
+                _control.txtLookFor.Text = lookFor;
+                _control.chkMatchCase.Checked = matchCase;
+                _control.chkMatchWholeWord.Checked = matchWholeWordOnly;
+
+                return this;
+            }
+
+            public TestAccessor SetEditor(TextEditorControl textEditorControl)
+            {
+                _control.Editor = textEditorControl;
+
+                return this;
+            }
+
+            public TestAccessor SetScanRegion(ISelection selection)
+            {
+                _control._search.SetScanRegion(selection);
+
+                return this;
+            }
+
+            public TestAccessor SetFileLoader(GetNextFileFnc fileLoader)
+            {
+                _control.SetFileLoader(fileLoader);
+
+                return this;
+            }
```

Comment:
This doesn't look needed, `SetFileLoader` is marked as `internal` so it should be accessible in tests directly.

Reply:
Removed.